tests/manager/manager-ctor-driver-metadata-001.phpt

```diff
@@ -1,5 +1,13 @@
 --TEST--
 MongoDB\Driver\Manager: Pass custom handshake data
+--DESCRIPTION--
+This test matches spaces at the end of the handshake data it appends. The only
+way to see final output is by checking against the binary socket communication:
+[2021-02-17T13:57:57.155166+00:00]     socket: TRACE   > 00100:  66 6f 72 6d 00 76 00 00  00 50 48 50 20 37 2e 34  f o r m . v . .  . P H P   7 . 4
+[2021-02-17T13:57:57.155182+00:00]     socket: TRACE   > 00110:  2e 31 35 20 2f 20 6d 69  6e 65 20 63 66 67 3d 30  . 1 5   /   m i  n e   c f g = 0
+
+Since matching this is not trivial, we're happy matching the trailing space at
+the end of each handshake data item.
```

Comment:
The trailing space will ultimately be removed once you address a related ticket in libmongoc, correct?

Reply:
Yes, it can be removed once we fix that.